src/psbtv2.ts

```diff
@@ -0,0 +1,910 @@
+import { BufferReader, BufferWriter } from 'bufio';
```

Comment:
Might be nice to link to the BIP for reference at the top of the file: https://github.com/bitcoin/bips/blob/master/bip-0370.mediawiki

Reply:
And then for the various consts and enums that are dictated by the BIP itself we can similarly indicate that they should match the respective section of the BIP